codes.json

```diff
@@ -878,21 +878,21 @@
   ,
   {
     "title" : "Smilei",
-    "subtitle" : "TBD/",
-    "description" : "TBD/",
-    "application": ["laser-plasma"],
-    "compute": ["mpi", "openmp"],
-    "method": ["pic"],
-    "phenomena": [],
-    "homepage" : "http://www.maisondelasimulation.fr/projects/Smilei/html/licence.html",
-    "institutions" : ["TBD/"],
-    "institution_link" : ["TBD/"],
-    "repository" : "TBD/",
-    "issue_tracker" : "TBD/",
-    "platform": [],
+    "subtitle" : "A high-performance, user-friendly electromagnetic Particle-In-Cell code.",
+    "description" : "Smilei is an electromagnetic Particle-In-Cell code offering high performance, user friendliness and a large range of applications. It includes many physics modules (ionisation, collisions, QED, ...) and advanced numerical techniques (cylindrical geometry, envelope solver, perfectly-matched layers, ...)",
+    "homepage" : "https://smileipic.github.io/Smilei",
+    "application": ["laser-plasma", "astrophysics", "space physics"],
+    "compute": ["C++", "python", "MPI", "openmp", "SIMD", "GPU"],
```

Comment:
Oh, is GPU support already released?

Reply:
Actually not officially. You are correct. I will remove it, then add it when it is released.

Reply:
Oh, ok. Yes update anytime :)